.github/workflows/rit.yml

```diff
@@ -0,0 +1,68 @@
+name: Rootstock Integration Tests
+
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches:
+      - "**"
+  push:
+    branches: ["master", "*-rc"]
+  workflow_dispatch:
+    inputs:
+      rit-branch:
+        description: 'Branch for Rootstock Integration Tests'
+        required: false
+        default: 'main'
+      powpeg-branch:
+        description: 'Branch for PowPeg Node'
+        required: false
+        default: 'master'
+
+jobs:
+  rootstock-integration-tests:
+    name: Rootstock Integration Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Set Branch Variables
+        id: set-branch-variables
+        run: |
+          # Default values
+          RSKJ_BRANCH="master"
+          RIT_BRANCH="${{ github.event.inputs.rit-branch || 'main' }}"
+          POWPEG_BRANCH="${{ github.event.inputs.powpeg-branch || 'master' }}"
+
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            RSKJ_BRANCH="${{ github.event.pull_request.head.ref }}"
+          elif [[ "${{ github.event_name }}" == "push" ]]; then
+            if [[ "${{ github.ref }}" == refs/heads/*-rc ]]; then
+              RSKJ_BRANCH="${{ github.ref }}"
+            fi
```

Comment:
question:

This condition `"${{ github.ref }}" == refs/heads/*-rc` can be added as an AND in the previous if couldn't it? If yes, it will make it simpler, I suggest to add it there.

Reply:
Good catch! solved